production-swarm/docker-stack.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+services:
+  jobservice:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env
+      - ./jobservice.env
+    environment:
+      CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
+      CAF_DATABASE_PASSWORD: "${CAF_DATABASE_PASSWORD:-root}"
+      CAF_STATUS_CHECK_TIME: 5
+      CAF_TRACKING_PIPE: "${CAF_TRACKING_PIPE_IN:-jobtracking-in}"
```

Comment:
These are not configurable in marathon I see.  I'm happy to make them none configurable here also